Views/Fields/TaxonomyField-Category.liquid

```diff
@@ -1,7 +1,15 @@
 ﻿<span class="taxonomy-field">
     {{ Model.ContentField.DisplayName }}
-    
+
+    {% assign termCount = Model.Terms | Size %}
+
     {% for term in Model.Terms %}
-        <a href="{% Href term.ContentItem.AutoroutePart.DisplayAlias %}">{{ term.Name }}</a>
+        {% assign termCount = termCount | Minus: 1 %}
+        
+        {% if termCount > 0 %}
+            <a href="{% Href term.ContentItem.AutoroutePart.DisplayAlias %}">{{ term.Name }},</a>

```

Comment:
Most miért része a vessző a linknek?

Reply:
Ez csak egy ötlet volt, de így ugyan azok a stylingok vonatkoznak rá, mint a linkre, és így látszik... De ha ez nem szép így, akkor átírom.

Reply:
Nem igazán, a vesszőnek magának nem kellene a link részének lennie. De a CSS-ben könnyen átírhatod a selectort, és akkor ugyanaz fog vonatkozni a vesszőre is.